magento/magento2
# Feature south america country regions

### Description (*)
Add states/regions for Argentina, Bolivia, Chile, Ecuador, Guyana, Paraguay, Peru, Suriname and Venezuela.


### Manual testing scenarios (*)

1. Install Magento
2. Check table directory_country_region for Argentina, Bolivia, Chile, Ecuador, Guyana, Paraguay, Peru, Suriname and Venezuela states/regions.

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#31169: Feature south america country regions
